index.html

```diff
@@ -1,52 +1,352 @@
+                            <div class="card-header">
+                                <h4 class="card-title">Multi-Post Stories Gain+Glory</h4>
+                            </div>
+                            <div class="card-body">
+                                <div class="language card-ruby">
+                                    <a href="http://">Ruby on Rails</a>
+                                </div>
+                                <div class="language card-css">
+                                    <a href="http://">Css</a>
+                                </div>
+                                <div class="language card-js">
+                                    <a href="http://">JavaScript</a>
+                                </div>
+                                <div class="language card-html">
+                                    <a href="http://">HTML</a>
+                                </div>
+                            </div>
+                            <div class="card-see">
+                                <div class="see-project">
+                                    <a href="http://#"> See Project</a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="cards">
+                    <div class="card-headerbg">
+                    </div>
+                    <div class="card">
+                        <div class="card-gra">
+                            <div class="card-header">
+                                <h4 class="card-title">Multi-Post Stories Gain+Glory</h4>
+                            </div>
+                            <div class="card-body">
+                                <div class="language card-ruby">
+                                    <a href="http://">Ruby on Rails</a>
+                                </div>
+                                <div class="language card-css">
+                                    <a href="http://">Css</a>
+                                </div>
+                                <div class="language card-js">
+                                    <a href="http://">JavaScript</a>
+                                </div>
+                                <div class="language card-html">
+                                    <a href="http://">HTML</a>
+                                </div>
+                            </div>
+                            <div class="card-see">
+                                <div class="see-project">
+                                    <a href="http://#"> See Project</a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+        </section>
+
+        <section class="about">
+
+
+            <div class="about-header">
+                <h1>About me</h1>
```

Comment:
- [ ]  kindly consider giving this header another hx like h2 because using only one H1 creates hierarchy and gives your page a more logical structure